client/metric_dashboards.go

```diff
@@ -57,6 +58,14 @@ type Label struct {
 	Value string `json:"label_value"`
 }
 
+type Panel struct {
```

Comment:
this matches the publicapi implementation here

https://github.com/lightstep/lightstep/blob/1f3b9c5eabb01ab1061810fe95f6b86b969199ed/go/src/github.com/lightstep/crouton/service/api/public/v02/handlers/metric_dashboards.go#L150-L163